src/view/items/shared/EventDirective.js

```diff
@@ -121,9 +122,14 @@ export default class EventDirective {
 
 			// Auto prevent and stop if return is explicitly false
 			let original;
-			if ( result === false && event && ( original = event.original ) ) {
-				original.preventDefault && original.preventDefault();
-				original.stopPropagation && original.stopPropagation();
+
+			if ( result === false ) {
+				if ( event && ( original = event.original ) ) {
+					original.preventDefault && original.preventDefault();
+					original.stopPropagation && original.stopPropagation();
+				} else {
+					warnIfDebug( 'Handler returns false, but no event object is available' );
+				}
```

Comment:
Thanks for your feedback, which I totally agree with. I was trying to add the warning in a minimally invasive way for easy reviewability. Went with your straightforward first suggestion.

BTW: One might consider enabling in eslint: no-cond-assign (though it wouldn't have caught this one) and prefer-const;)